Assets/Scripts/Text/ConversationTextManager.cs

```diff
@@ -172,26 +158,49 @@ private void ChangeLine(int increase)
         mainTextDrawer.InitializeLine();
     }
 
-    private void ChangeFlag()
+    private void ChangeQuestionData()
     {
-        if (talkData.Content[lineNumber].QuestionData == null)
-            return;
-        if (talkData.Content[lineNumber].QuestionData[question.GetCursorPlace()].NextFlag == null)
-            return;
+        QuestionData[] questionData = talkData.Content[lineNumber].QuestionData;
+        if (questionData != null)
+        {
+            QuestionData talkDataContent = questionData[question.GetCursorPlace()];
+            if (talkDataContent.NextFlag != null)
+            {
+                ChangeFlag(talkDataContent.NextFlag);
+            }
+        }

```

Comment:
```suggestion
        if (questionData == null) return;
        var nextFlag = questionData[question.GetCursorPlace()].NextFlag;
        if (nextFlag == null) return;
        ChangeFlag(nextFlag);
```
早期リターンでネストを少なくして読みやすくしてほしい。あと今回はこれ以降talkDataContentのNextFlagしか使わないので、そこまで変数に入れて良さそう。